volumina/widgets/multiscaleFileExportOptionsWidget.py

```diff
@@ -0,0 +1,134 @@
+###############################################################################
+#   volumina: volume slicing and editing library
+#
+#       Copyright (C) 2011-2025, the ilastik developers
+#                                <team@ilastik.org>
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the Lesser GNU General Public License
+# as published by the Free Software Foundation; either version 2.1
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Lesser General Public License for more details.
+#
+# See the files LICENSE.lgpl2 and LICENSE.lgpl3 for full text of the
+# GNU Lesser General Public License version 2.1 and 3 respectively.
+# This information is also available on the ilastik web site at:
+# 		   http://ilastik.org/license/
+###############################################################################
+import os
+
+from PyQt5 import uic
+from PyQt5.QtCore import pyqtSignal, Qt, QEvent
+from PyQt5.QtWidgets import QWidget, QFileDialog
+
+
+class MultiscaleFileExportOptionsWidget(QWidget):
+    def __init__(self, parent):
+        super().__init__(parent)
+        uic.loadUi(os.path.splitext(__file__)[0] + ".ui", self)
+        self.file_extensions = (".zarr",)
+        self.default_extension = ".zarr"
+        self.extension_description = "Zarr files (*.zarr)"
+
+        # We need to watch the textEdited signal because Qt has a bug that causes the OK button
+        #  to receive it's click event BEFORE the LineEdit receives its FocusOut event.
+        # (That is, we can't just watch for FocusOut events and disable the button before the click.)
```

Comment:
Does this still hold true? That comment in it's original form from the hierarchical export widget is from 2013...